UI/data/locale/en-US.ini

```diff
@@ -927,6 +927,15 @@ Basic.Settings.Stream.Recommended.MaxFPS="Maximum FPS: %1"
 # basic mode 'output' settings
 Basic.Settings.Output="Output"
 Basic.Settings.Output.Format="Recording Format"
+Basic.Settings.Output.Format.MKV="Matroska Video (.mkv)"
+Basic.Settings.Output.Format.FLV="Flash Video (.flv)"
+Basic.Settings.Output.Format.MP4="MPEG-4 (.mp4)"
+Basic.Settings.Output.Format.MOV="QuickTime (.mov)"
+Basic.Settings.Output.Format.TS="MPEG-TS (.ts)"
+Basic.Settings.Output.Format.HLS="HLS (.m3u8 + .ts)"
+Basic.Settings.Output.Format.fMP4="Fragmented MP4 (.mp4)"
+Basic.Settings.Output.Format.fMOV="Fragmented MOV (.mov)"
+Basic.Settings.Output.Format.TT="Fragmented MP4/MOV writes the recording in chunks and does not require the same finalization as traditional MP4/MOV files.\nThis ensures the file remains playable even if writing to disk is interrupted e.g. as a result of a BSOD or power loss.\n\nThis may not be compatible with all players and editors. Use File → Remux Recordings to convert the file into a more compatible format if necessary."
```

Comment:
"e.g." should always be followed by a comma ("e.g.,").
Parenthesis are not required, but may be clearer:
> This ensures the file remains playable even if writing to disk is interrupted (e.g., as a result of a BSOD or power loss.)

Otherwise, a comma may be encourage:
> This ensures the file remains playable even if writing to disk is interrupted, e.g., as a result of a BSOD or power loss.